po/es.po

```diff
@@ -216,30 +167,431 @@ msgstr "_Acerca de Eyedropper"
+msgid "F11 (Ultralume 40, Philips TL84)"
+msgstr "F11 (Ultralume 40, Philips TL84)"
+
+#: data/resources/ui/preferences.ui:69
+msgctxt ""
+"CIE illuminant, more info at https://en.wikipedia.org/wiki/"
+"Standard_illuminant"
+msgid "F12 (Ultralume 30, Philips TL83)"
+msgstr "F12 (Ultralume 30, Philips TL83)"
+
+#: data/resources/ui/preferences.ui:77
+msgid "Name Sets"
+msgstr "Conjuntos de nombres"
+
+#: data/resources/ui/preferences.ui:78
+msgid "Which color name sets should be used for the names"
+msgstr "Qué conjuntos de nombres de colores deben utilizarse para los nombres"
+
+#: data/resources/ui/preferences.ui:92
+msgid "Color Formats"
+msgstr "Formatos de color"
+
+#: data/resources/ui/preferences.ui:93
+msgid "Customize the visible formats and in which order they are displayed."
+msgstr "Personaliza los formatos visibles y el orden en que se muestran."
+
+#: data/resources/ui/preferences.ui:97
+msgctxt "Tooltip of the button to reset the format order"
+msgid "Reset Order"
+msgstr "Restablecer orden"
+
+#: data/resources/ui/color-format-row.ui:22
+msgctxt "Tooltip of the copy button for format rows"
+msgid "Copy Format"
+msgstr "Copiar formato"
+
+#: data/resources/ui/palette-window.ui:5
+msgid "Palettes"
+msgstr "Paletas"
 
 #. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
-#: src/application.rs:167
+#: src/widgets/about_window.rs:50
 msgid "translator-credits"
-msgstr "Sergio Varela https://github.com/IngrownMink4"
-
-#: src/widgets/preferences.rs:214
+msgstr "Sergio Varela <https://github.com/IngrownMink4>"
+
+#: src/widgets/about_window.rs:110
+msgctxt ""
+"The changelog of the current version. Should be similar to the CHANGELOG.md "
+"file at the project root. Please ensure that the tags have a matching "
+"closing tag. Since xgettext does not recognize rust multiline strings, this "
+"should be on a single line using line breaks (\\n) for new lines."
+msgid ""
+"<p>This release extends the current available features.</p><p> Features:</"
+"p>\n"
+"<ul><li> The name of the color, specified by w3c color keywords and the xkcd "
+"color survey, can now be shown</li><li>Switch the color by entering a color "
+"keyword</li><li> A new Palette dialog showing different color palettes</"
+"li><li>Choose which CIE standard observer is used, as well as it's "
+"illuminant</li><li> Show the CIELCh color format</li><li> Using the new "
+"AdwAboutWindow</li><li> Added new translations</li><li>Fixed minor issues</"
+"li></ul>"
+msgstr ""
+"<p>Esta versión amplía las funciones disponibles actualmente.</p><p> Características:"
+"</p>\n"
+"<ul><li> El nombre del color, especificado por las palabras clave de color w3c y la encuesta de color xkcd "
+", puede mostrarse ahora</li><li>Cambie el color introduciendo un color "
+", palabra clave</li><li>Un nuevo cuadro de diálogo de la paleta que muestra diferentes paletas de colores</"
```

Comment:
I have no idea of Spanish punctuation rules, but this would be rendered with a space before the comma (`xkcd , puede`). Is that intended? Same thing on the next line with `color , palabra`.